stac_geoparquet/stac_geoparquet.py

```diff
@@ -24,7 +30,9 @@ def _fix_array(v):
     return v
 
 
-def to_geodataframe(items: Sequence[dict[str, Any]]) -> geopandas.GeoDataFrame:
+def to_geodataframe(
+    items: Sequence[dict[str, Any]], add_self_link: bool = False
```

Comment:
Let's align this new parameter name with the name of the column (link vs. href).